README.md

```diff
@@ -12,206 +12,614 @@ A command-line tool for managing TEE deployments on Phala Network, from local de
+```
+
+### TEE Simulator Commands
+
+Commands for managing the local TEE simulator for development and testing.
+
+#### Start Simulator
+
+```bash
+phala simulator start [options]
+```
+
+Start the TEE simulator locally for development and testing.
+
+**Options:**
+- `-i, --image <image>`: Simulator image (defaults to 'phalanetwork/phala-pruntime:latest')
+
+**Example:**
+```bash
+phala simulator start
+```
+
+#### Stop Simulator
+
+```bash
+phala simulator stop
+```
+
+Stop the running TEE simulator.
+
+**Example:**
+```bash
+phala simulator stop
+```
+
+### Configuration Commands
+
+Commands for managing CLI configuration settings.
+
+#### Get Configuration Value
+
+```bash
+phala config get <key>
+```
+
+Get a specific configuration value.
+
+**Arguments:**
+- `key`: Configuration key to retrieve
+
+**Example:**
+```bash
+phala config get apiUrl
+```
+
+#### Set Configuration Value
+
+```bash
+phala config set <key> <value>
+```
+
+Set a configuration value.
+
+**Arguments:**
+- `key`: Configuration key to set
+- `value`: Value to set (can be a string, number, boolean, or JSON)
+
+**Example:**
+```bash
+phala config set defaultVcpu 2
+phala config set apiUrl "https://custom-api.phala.cloud"
+phala config set debug true
+phala config set customConfig '{"key": "value", "nested": {"array": [1, 2, 3]}}'
+```
+
+#### List Configuration Values
+
+```bash
+phala config list [options]
+```
+
+List all configuration values.
+
+**Options:**
+- `-j, --json`: Output in JSON format
+
+**Example:**
+```bash
+phala config list
+phala config list --json
+```
+
+### Cloud Virtual Machine (CVM) Commands
+
+Commands for managing Cloud Virtual Machines (CVMs) on Phala Cloud.
+
+#### List CVMs
+
+```bash
+phala cvms list [options]
+```
+
+List all CVMs associated with your account.
+
+**Options:**
+- `-j, --json`: Output in JSON format
+
+**Example:**
+```bash
+phala cvms list
+```
+
+#### Get CVM Details
+
+```bash
+phala cvms get [options] <app-id>
+```
+
+Get detailed information about a specific CVM.
+
+**Arguments:**
+- `app-id`: App ID of the CVM
+
+**Options:**
+- `-j, --json`: Output in JSON format
+
+**Example:**
+```bash
+phala cvms get app_123456
+```
+
+#### Create CVM
+
+```bash
+phala cvms create [options]
+```
+
+Create a new CVM on Phala Cloud.
+
+**Options:**
+- `-n, --name <name>`: Name of the CVM (required)
+- `-c, --compose <compose>`: Path to Docker Compose file (required)
+- `-t, --type <type>`: Type of CVM (default: 'phala')
+- `-m, --mode <mode>`: Mode of operation (default: 'docker-compose')
+- `--vcpu <vcpu>`: Number of vCPUs (default: 1)
+- `--memory <memory>`: Memory in MB (default: 2048)
+- `--disk-size <diskSize>`: Disk size in GB (default: 20)
+- `-e, --env <env...>`: Environment variables in the form of KEY=VALUE
+- `--env-file <envFile>`: Path to environment file
+- `--debug`: Enable debug mode
+
+**Example:**
+```bash
+phala cvms create --name my-tee-app --compose ./docker-compose.yml --vcpu 2 --memory 4096 --env-file ./.env
+```
```

Comment:
Now that the user have created their first CVM, we need to make them feel the TEE-proof maybe?

Reply:
I added a `phala cvms attestation` command for fetching the attestation.